assets/utils.js

```diff
@@ -753,3 +753,19 @@ export function isActionEnabled(configKey) {
     const config = getConfig(configKey, {});
     return (action) => config[action.id] == null || config[action.id];
 }
+
+function convertHtmlToPlainText(html) {
```

Comment:
I think this conversion has an issue for markup like:
```
<p>one</p><p>two</p>
```
will generate `onetwo`. not sure if that could be an issue in this case, but if we might be dealing with ingested content probably yes. there is this helper `getTextFromHtml` which handles it.